docs/src/kubectl-plugin.md

```diff
@@ -1063,6 +1063,216 @@ kubectl cnpg pgadmin4 --dry-run cluster-example | kubectl delete -f -
+```
+
+### Logical Replication Subscriptions
+
+The `cnpg subscription` command group is a dedicated set of commands designed
+to simplify the creation and removal of [PostgreSQL logical replication
+subscriptions](https://www.postgresql.org/docs/current/logical-replication-subscription.html).
+These commands are specifically crafted to aid in the establishment of logical
+replication subscriptions, especially when dealing with remote PostgreSQL
+databases.
+
+!!! Warning
+    Before utilizing these commands, it is essential to have a comprehensive
+    understanding of both the capabilities and limitations of PostgreSQL's native
+    logical replication system.
+
+In addition to subscription management, we provide a helpful command for
+synchronizing all sequences from the source. While its applicability may vary,
+this command can be particularly useful in scenarios involving major upgrades
+or data import from remote servers.
+
+#### Creating a new subscription
+
+To create a logical replication subscription, use the `cnpg subscription create`
+command. The basic structure of this command is as follows:
+
+```sh
+kubectl cnpg subscription create \
+  --subscription <SUBSCRIPTION_NAME> \
+  --publication <PUBLICATION_NAME> \
+  --external-cluster <EXTERNAL_CLUSTER> \
+  <LOCAL_CLUSTER> [options]
+```
+
+This command configures a subscription directed towards the specified
+publication in the designated external cluster, as defined in the
+`<LOCAL_CLUSTER>`.
+
+For additional information and detailed instructions, type the following
+command:
+
+```sh
+kubectl cnpg subscription create help
+```
+
+!!! Warning
+    Prioritize testing subscriptions in a non-production environment to ensure
+    their effectiveness and identify any potential issues before implementing them
+    in a production setting.
+
+#### Dropping a subscription
+
+The `cnpg subscription drop` command seamlessly complements the `create` command.
+You can drop a `SUBSCRIPTION` with the following command structure:
+
+```sh
+kubectl cnpg subcription drop \
+  --subscription <SUBSCRIPTION_NAME> \
+  <LOCAL_CLUSTER> [options]
+```
+
+To access further details and precise instructions, use the following command:
+
+```sh
+kubectl cnpg subscription drop help
+```
+
+#### Synchronizing sequences
+
+One of the limitations of PostgreSQL logical replication built on top of
+publication and subscriptions is that sequences are not synchronized.
+
+However, often logical replication is used to migrate a live database on
+another one, even on an higher version of PostgreSQL. Before cutting over
+applications to the new database, sequences are updated.
+The `cnpg subscription sync-sequences` command connects to the database of
+source, pulls down all the sequences, and then updates the ones with the same
+identity (database schema and sequence name) locally.
+
+The command to execute is similar to the following:
+
+```sh
+kubectl-cnpg subscription sync-sequences \
```

Comment:
```suggestion
kubectl cnpg subscription sync-sequences \
```

Reply:
Actually, the entire paragraph is repeated twice

Reply:
You are right. Let me fix it.

Reply:
fixed